deeprankcore/domain/edgefeatures.py

```diff
@@ -1,3 +1,10 @@
+## main group
+EDGE = "edge_features"
+
+## metafeatures
+NAME = "_name"
+INDEX = "_index"
```

Comment:
What are the main group and metafeatures for?

Reply:
main group indicates the hdf5 Group name, containing the hdf5 Datasets whose names are listed below. In edges case it's called `edge_features`, and we have the same structure for nodes. Metafeatures are the Datasets' names used for identifying the edges that are not used as features for training the networks.

Reply:
I see, they're not really feature names. They indicate what the hdf5 groups should be named. I think it's a bit confusing to have this module named  edgefeatures. Perhaps it can be `hdf5/storage/features.py` or similar.

Reply:
I think it's fine that they are in the domain/ folder because they contain domain-related features (both for edges and nodes). Also, it's good to minimize the number of subtrees in the package and try to find a tradeoff between clearness, readability, and simplicity. What about `domain/edgestorage.py` and `domain/nodestorage.py`? @cbaakman @DaniBodor

Reply:
I think that would do fine.

Reply:
Done, now we have edgestorage.py, nodestorage.py and targetstorage.py files containing the global variables used for naming Groups and Datasets in the hdf5 files

Reply:
i also like this: 

> What about `domain/edgestorage.py` and `domain/nodestorage.py`?